lib/mongo/collection/view.rb

```diff
@@ -131,6 +131,8 @@ def hash
       # @option options :sort [ Hash ] The key and direction pairs used to sort the
       #   results.
       # @option options [ Hash ] :collation The collation to use.
+      # @option options [ Array ] :array_filters A set of filters specifying to which array elements
+      #   an update should apply.
```

Comment:
You don't need to document ``array_filters`` here, as the methods ``update_one``, ``update_many``, ``find_one_and_replace``, ``find_one_and_update`` pass the options from API method on collection to the corresponding method in writable.

For example, calling ``collection.update_many({..}, {..}, array_filters => [..])``, creates a ``Collection::View``, then immediately calls ``#update_many`` on the view. It passes the options directly to ``Collection::View::Writable``
[see here](https://github.com/mongodb/mongo-ruby-driver/blob/master/lib/mongo/collection.rb#L513)

Let me know if that's not clear.